install/kubernetes/cilium/charts/nodeinit/templates/daemonset.yaml

```diff
@@ -81,6 +81,19 @@ spec:
                     systemctl restart kubelet
 {{- end }}
 
+{{- if .Values.global.gke.enabled }}
+                    # If the IP-MASQ chain exists, add back jump rule
```

Comment:
This needs a more descriptive comment on why this is needed instead of what the code does. Why is this needed? Why does Cilium need to manipulate k8s iptables rules?

Reply:
@tgraf Add more info to the comment in the install script portion